nilearn/datasets.py

```diff
@@ -204,39 +206,43 @@ def _get_dataset_dir(dataset_name, data_dir=None, env_vars=[],
     -----
     This function retrieves the datasets directory (or data directory) using
     the following priority :
-    1. the keyword argument data_dir
-    2. the global environment variable NILEARN_SHARED_DATA
-    3. the user environment variable NILEARN_DATA
-    4. nilearn_data in the user home folder
+    1. defaults system paths
+    2. the keyword argument data_dir
+    3. the global environment variable NILEARN_SHARED_DATA
+    4. the user environment variable NILEARN_DATA
+    5. nilearn_data in the user home folder
```

Comment:
Can you amend doc/manipulating_visualizing/manipulating_images.rst which has the same description (or a very similar one)?